src/talos/cpg_internal_scripts/annotation_stages.py

```diff
@@ -247,28 +249,3 @@ def queue_jobs(self, cohort: targets.Cohort, inputs: stage.StageInput) -> stage.
         )
 
         return self.make_outputs(cohort, data=output, jobs=job)
-
-
-@stage.stage(
-    analysis_type='talos_prep',
-    required_stages=[JumpAnnotationsFromHtToFinalMtStage],
-)
-class SquashMtIntoTarballStage(stage.CohortStage):
```

Comment:
No need to compress the annotation outputs, this saves ~40 mins per run at large data sizes